src/P6/mapper.py

```diff
@@ -71,24 +72,71 @@ def apply_mapping(
     "denovo": "de_novo_mutation",
 }
 
+# Check Variant column groups: allow either the full raw coordinates or any HGVS notation
+RAW_VARIANT_COLUMNS = {
+    "chromosome",
+    "start_position",
+    "end_position",
+    "reference",
+    "alternate",
+}
+HGVS_VARIANT_COLUMNS = {"hgvsg", "hgvsc", "hgvsp"}
+# minimal base columns to call something a genotype sheet (we bring the index in later)
+GENOTYPE_BASE_COLUMNS = {"contact_email", "phasing"}
+
+
+class TableMapper(metaclass=abc.ABCMeta):
+    @abc.abstractmethod
+    def apply_mapping(
+        self, tables: dict[str, pd.DataFrame], notepad: Notepad
+    ) -> typing.Sequence[Phenopacket]:
+        pass
+
 
 class DefaultMapper(TableMapper):
-    def __init__(self, hpo: hpotk.MinimalOntology):
+    def __init__(self, hpo: hpotk.MinimalOntology, strict_variants: bool = False):
+        """
+        - False: raw⇄HGVS mismatches are logged as WARNINGS
+        - True : raw⇄HGVS mismatches are logged as ERRORS
+        """
         self._hpo = hpo
+        self.strict_variants = strict_variants
 
     def apply_mapping(
         self, tables: dict[str, pd.DataFrame], notepad: Notepad
-    ) -> tuple[list[Genotype], list[Phenotype]]:
+    ) -> tuple[
```

Comment:
I see a lint in my IDE - the method does not override the abstract method in the correct fashion.

It should return a `typing.Sequence[Phenopacket]`, but it returns a tuple of other things.

We should either change the `TableMapper`s signature or return a sequence of phenopackets from the overriding method (this method, preferred solution).